apine.schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#", 
+  "$id": "https://github.com/INCF/apine/blob/master/apine.schema.json", 
+  "type": "object", 
+  "additionalProperties": false, 
+  "properties": {
+    "dataset": {
+      "$id": "https://github.com/INCF/apine/blob/master/apine.schema.json/dataset", 
+      "minLength": 1,
+      "type": "string", 
+      "title": "dataset", 
+      "description": "The parent" 
```

Comment:
how about assigning uris to datasets?

Reply:
@glatard great point! I've restricted the field to be a `uri-reference`